specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/botservice.json

```diff
@@ -2127,6 +2416,122 @@
           "type": "string"
         }
       }
+    },
+    "EnterpriseChannelCheckNameAvailabilityRequestBody": {
+      "description": "The request body for a request to Bot Service Management to check availability of an Enterprise Channel name.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "the name of the Enterprise Channel for which availability needs to be checked.",
+          "type": "string"
+        }
+      }
+    },
+    "EnterpriseChannelCheckNameAvailabilityResponseBody": {
+      "description": "The response body returned for a request to Bot Service Management to check availability of an Enterprise Channel name.",
+      "type": "object",
+      "properties": {
+        "valid": {
+          "description": "indicates if the Enterprise Channel name is valid.",
+          "type": "boolean"
+        },
+        "message": {
+          "description": "additional message from the bot management api showing why a bot name is not available",
```

Comment:
Minor grammar: capital 'A' and add period '.'

Also, in other places you have capitalized "Bot Service Management" - I would probably just simplify the description, e.g.:
"Additional information about why a bot name is not available."